src/Microsoft.Graph/Constants.cs

```diff
@@ -77,5 +77,21 @@ public static class OdataInstanceAnnotations
             /// </summary>
             public const string DeltaLink = "@odata.deltaLink";
         }
+
+        /// <summary>
+        /// Constants used to specify content types
+        /// </summary>
+        public static class ContentTypes
```

Comment:
Can we move this to core so that it is available to beta service library as well?

Reply:
Makes sense. Will action and resolve.

Reply:
We can actually take advantage of values already here.
https://github.com/microsoftgraph/msgraph-sdk-dotnet-core/blob/c914abb65a3c0e388166fd26f21d187325180da2/src/Microsoft.Graph.Core/CoreConstants.cs#L58

Reply:
Resolved via https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/905/commits/3c46cb40e8c3f9be4c2d2a4360ee34d002357d6a